src/Compilers/CSharp/csc/AnyCpu/csc.csproj

```diff
@@ -3,8 +3,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net6.0;net472</TargetFrameworks>
-    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">net8.0;$(TargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(SourceBuildTargetFrameworks);net472</TargetFrameworks>
```

Comment:
I wonder whether we should be building net472 for source-build at all?

Reply:
You mean to speed the build up? Then we don't have to build net6.0 either, only net8.0?

Reply:
Yeah. Especially the net6.0 targets, I can only assume that this will cause pain in source-build. If they could easily build net6.0, we wouldn't need to update in the first place.